src/library/handlers/logHandler.ts

```diff
@@ -330,6 +322,16 @@ export class LogHandler {
             }
             const serviceInstanceId = attributes[SemanticResourceAttributes.SERVICE_INSTANCE_ID];
             tags[KnownContextTagKeys.AiCloudRoleInstance] = String(serviceInstanceId);
+            tags[KnownContextTagKeys.AiInternalSdkVersion] = resourceManager.getInternalSdkVersion();
+        }
+        // Add Correlation headers
+        const spanContext = trace.getSpanContext(context.active());
+        if (spanContext) {
+            tags[KnownContextTagKeys.AiOperationId] = spanContext.traceId;
+            tags[KnownContextTagKeys.AiOperationParentId] = spanContext.spanId;
+        }
+        else{
+            tags[KnownContextTagKeys.AiOperationId] = this._idGenerator.generateTraceId();
```

Comment:
Python sets trace_Id to 0 if not present in SpanContext. Not sure if this what we want (because this behavior is not well-defined), but we should probably be consistent.

Reply:
The JS Sdk generates a new one (like this) when the provided trace id is not provided or invalid (when it's being set).

Reply:
is there anything special that happens on Breeze side with the 0 value?

Reply:
I do not believe so, just know that this is how Opencensus is doing this. Perhaps generating a new one is the way to go but am not certain of the implications.